docs/compatibility.md

```diff
@@ -8,9 +8,9 @@ nav_order: 6
 
 ## Ruby & Rails
 
-ViewComponent is [supported natively](https://edgeguides.rubyonrails.org/layouts_and_rendering.html#rendering-objects) in Rails 6.1, and compatible with Rails 5.2+ via an included [monkey patch](https://github.com/viewcomponent/view_component/blob/main/lib/view_component/render_monkey_patch.rb).
+ViewComponent supports all actively supported versions of Ruby (3.0+) and Ruby on Rails(6.1+) and is tested against a combination of these versions of Ruby on Rails.
 
-ViewComponent is tested for compatibility [with combinations of](https://github.com/viewcomponent/view_component/blob/22e3d4ccce70d8f32c7375e5a5ccc3f70b22a703/.github/workflows/ruby_on_rails.yml#L10-L11) Ruby v2.5+ and Rails v5.2+. Ruby 2.4 is likely compatible, but is no longer tested.
+While EOL (end-of-life) versions of Ruby (>= 2.7) and Ruby on Rails (> 6.0) may still work with ViewComponent, they are not actively supported and we do not active test the gem against them. We will still accept patches on a case-by-case basis to support older Ruby and Ruby on Rails versions based on the complexity and maintenance burden. Please open an issue before submitting a Pull Request that addresses issues with EOL Ruby/Ruby on Rails versions.
```

Comment:
```suggestion
While EOL (end-of-life) versions of Ruby and Ruby on Rails may still work with ViewComponent, they are not actively supported and no longer tested. We will still accept patches on a case-by-case basis to support older Ruby & Rails versions based on the complexity and maintenance burden. Please open an issue before submitting such a Pull Request.
```

I'm a bit reluctant to mention the specific EOL versions here since we'll need to keep that up-to-date.

Reply:
@Spone I agree with you, but we will still have to remove and add versions to CI when versions get released and EOL'd